pkg/chrootarchive/diff_unix.go

```diff
@@ -49,6 +51,10 @@ func applyLayer() {
 		fatal(err)
 	}
 
+	if inUserns {
+		options.InUserNS = true
```

Comment:
fwiw, I triggered this by running "sudo adt-run --unbuilt-tree .  --- null". and I just tried the same on arm64 with the same results, so it would probably happen on amd64 too...